Gopkg.toml

```diff
@@ -106,7 +106,7 @@ required = [
 ### DigitalOcean
 [[constraint]]
   name = "github.com/digitalocean/godo"
-  version = "1.1.0"
+  revision = "7249498ac084d6cc697aae29fb1108acbfba99e2"
```

Comment:
Hi @andrewsomething,

Please change this to:

```toml
[[constraint]]
    name = "github.com/digitalocean/godo"
    version = "1.2.0"
```

We prefer tagged releases, and the `github.com/digitalocean/godo@1.2.0` is only 10 days old. Please go ahead and rebase this PR onto `master` as well. Thank you!

Reply:
When first submitted, there had been no tagged release. Rebased and updated to use godo 1.2.0.

Thanks for the review!